app/graphql/resolvers/freak_create_resolver.rb

```diff
@@ -1,17 +1,21 @@
 # frozen_string_literal: true
 
+# TODO: Test if norm_id is not nil
```

Comment:
`norm_id` can never be nil due to the GraphQL type not allowing the user to pass a `null` value as an argument.

The todo should state that we need to validate whether the norm exists. Also, move it above the `resolve` method, since it's more related to it rather than the whole class.